examples/examples/http.rs

```diff
@@ -29,13 +29,41 @@ use std::time::Duration;
 
 use hyper::body::Bytes;
 use jsonrpsee::core::client::ClientT;
+use jsonrpsee::core::middleware::{Batch, Notification, RpcServiceBuilder, RpcServiceT};
 use jsonrpsee::http_client::HttpClient;
 use jsonrpsee::rpc_params;
 use jsonrpsee::server::{RpcModule, Server};
+use jsonrpsee::types::Request;
 use tower_http::LatencyUnit;
 use tower_http::trace::{DefaultMakeSpan, DefaultOnResponse, TraceLayer};
 use tracing_subscriber::util::SubscriberInitExt;
 
+struct Logger<S>(S);
```

Comment:
Clean! There is a Logger middleware implemented in the `rpc_middleware.rs` example too. What's the thinking abotu adding one here?

Reply:
my bad, let's revert this